docs/reference/builder.md

```diff
@@ -1244,6 +1244,29 @@ The `STOPSIGNAL` instruction sets the system call signal that will be sent to th
 This signal can be a valid unsigned number that matches a position in the kernel's syscall table, for instance 9,
 or a signal name in the format SIGNAME, for instance SIGKILL.
 
+## CMD and ENTRYPOINT relation
```

Comment:
not sure I personally like the word relation. maybe `interaction`? `relationship`? something else? maybe it's fine.